OutOfSchool/OutOfSchool.EmailSender/Quartz/EmailSenderJob.cs

```diff
@@ -45,7 +45,7 @@ public async Task Execute(IJobExecutionContext context)
             var subject = dataMap.GetString(EmailSenderStringConstants.Subject);
             var htmlContent = dataMap.GetString(EmailSenderStringConstants.HtmlContent);
             var plainContent = dataMap.GetString(EmailSenderStringConstants.PlainContent);
-            var expirationTime = dataMap.GetDateTime(EmailSenderStringConstants.ExpirationTime);
+            var expirationTime = DateTimeOffset.Parse(dataMap.GetString(EmailSenderStringConstants.ExpirationTime));
```

Comment:
i do agree with [sonar](https://sonarcloud.io/organizations/ita-social-projects/rules?open=csharpsquid%3AS6580&rule_key=csharpsquid%3AS6580) that a format provider might be good for maintainability and so we know we use Ukrainian/European datetime format.